scipy/stats/studentizedrange/studentized_range.cpp

```diff
@@ -0,0 +1,46 @@
+//Run "gcc -shared -fPIC -o studentized_range.so studentized_range.cpp" to compile
+#include <math.h>
+
+struct RangeArg {
+    double q;
+    double k;
+    double v;
+};
+
+//PDF approximation
+double Phi(double value) {
+   return 0.5 * erfc(-value * M_SQRT1_2);
+}
+
+//PDF approximation
+double phi(double z)
+{
+    static const double inv_sqrt_2pi = 0.3989422804014327;
+    return inv_sqrt_2pi * std::exp(-0.5f * z * z);
```

Comment:
```suggestion
    return inv_sqrt_2pi * ::exp(-0.5f * z * z);
```
MacOS doesn't seem to like trying to get `exp` from `std` namespace:
```
studentized_range.cpp:19:32: error: no member named 'exp' in namespace 'std'
    return inv_sqrt_2pi * std::exp(-0.5f * z * z);
```